### ISSUE TITLE
Embed timestamp doesn't work

### ISSUE THREAD

**Issue by uwx** [NaamloosDT/DSharpPlus#170]
To reproduce:
```cs
var embed = new DiscordEmbedBuilder()
  .WithDescription("It's lonely in here")
  .WithTimestamp(DateTime.Now);
channel.SendMessageAsync("The timestamp won't work", embed);
```

ps: credits to @armaa for finding this :v

**Comment by armaa** [comment]
Strange enough, it works if you put in a ulong date, i tried it with '99844490909448899'.
Works in both, 3.1.2 and 3.2.0 nightly build.

**Comment by uwx** [comment]
should be fix with
```cs
new JsonSerializerSettings() { DateFormatHandling = DateFormatHandling.IsoDateFormat, DateParseHandling = DateParseHandling.DateTimeOffset, DateTimeZoneHandling = DateTimeZoneHandling.RoundtripKind };
```
PS: i license this snippet of code under the GPL (Giant Penis License) or MPL (Micro Penis License) whichever you prefer

**Comment by armaa** [comment]
I just tested `DateTimeOffset`, and that one works fine. Using `DateTimeOffset.Now` gives the correct todays date in the timestamp.

**Comment by Emzi0767** [comment]
Fixed by 8dca9257ba7cef61249a183f97ad625baac192ae.
